{
  "uid" : "5788feeef3cc9b73",
  "name" : "Verify sizes of images in contributor cards in grids",
  "fullName" : "tests.contributors_page_test.TestContributorCardImages#test_cnp_04_02_verify_images_sizes_in_cards",
  "historyId" : "77bc5d9c5052de5e9564db64c06c3e71",
  "time" : {
    "start" : 1728610933686,
    "stop" : 1728610933686,
    "duration" : 0
  },
  "status" : "broken",
  "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x5570954ad02a <unknown>\n#1 0x5570951935e0 <unknown>\n#2 0x55709518b7eb <unknown>\n#3 0x55709517ba09 <unknown>\n#4 0x55709517d744 <unknown>\n#5 0x55709517bcbd <unknown>\n#6 0x55709517b52f <unknown>\n#7 0x55709517b422 <unknown>\n#8 0x557095179493 <unknown>\n#9 0x557095179ada <unknown>\n#10 0x557095195f41 <unknown>\n#11 0x5570952278c5 <unknown>\n#12 0x557095207b22 <unknown>\n#13 0x557095226d7d <unknown>\n#14 0x5570952078c3 <unknown>\n#15 0x5570951d56b3 <unknown>\n#16 0x5570951d668e <unknown>\n#17 0x557095477a2b <unknown>\n#18 0x55709547b9b1 <unknown>\n#19 0x557095464225 <unknown>\n#20 0x55709547c532 <unknown>\n#21 0x55709544938f <unknown>\n#22 0x55709549bf28 <unknown>\n#23 0x55709549c0f3 <unknown>\n#24 0x5570954abe7c <unknown>\n#25 0x7f2d7d494ac3 <unknown>",
  "statusTrace" : "driver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"2b0bebe0fc7829f874c2d38e54d6945f\")>\n\n    @pytest.fixture()\n    @allure.step(f'Open page: {MainPageLinks.URL_CONTRIBUTORS_PAGE}')\n    def contributors_page_open(driver):\n>       driver.get(MainPageLinks.URL_CONTRIBUTORS_PAGE)\n\ntests/conftest.py:28: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:353: in get\n    self.execute(Command.GET, {\"url\": url})\n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:344: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7f3bf8a1d430>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"unknown error\",\"message\":\"unknown error: net::ERR_CONNECTION_REFUSED\\\\n  ...\\\\n#23 0x55709549c0f3 \\\\u003Cunknown>\\\\n#24 0x5570954abe7c \\\\u003Cunknown>\\\\n#25 0x7f2d7d494ac3 \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\nE         (Session info: chrome=129.0.6668.89)\nE       Stacktrace:\nE       #0 0x5570954ad02a <unknown>\nE       #1 0x5570951935e0 <unknown>\nE       #2 0x55709518b7eb <unknown>\nE       #3 0x55709517ba09 <unknown>\nE       #4 0x55709517d744 <unknown>\nE       #5 0x55709517bcbd <unknown>\nE       #6 0x55709517b52f <unknown>\nE       #7 0x55709517b422 <unknown>\nE       #8 0x557095179493 <unknown>\nE       #9 0x557095179ada <unknown>\nE       #10 0x557095195f41 <unknown>\nE       #11 0x5570952278c5 <unknown>\nE       #12 0x557095207b22 <unknown>\nE       #13 0x557095226d7d <unknown>\nE       #14 0x5570952078c3 <unknown>\nE       #15 0x5570951d56b3 <unknown>\nE       #16 0x5570951d668e <unknown>\nE       #17 0x557095477a2b <unknown>\nE       #18 0x55709547b9b1 <unknown>\nE       #19 0x557095464225 <unknown>\nE       #20 0x55709547c532 <unknown>\nE       #21 0x55709544938f <unknown>\nE       #22 0x55709549bf28 <unknown>\nE       #23 0x55709549c0f3 <unknown>\nE       #24 0x5570954abe7c <unknown>\nE       #25 0x7f2d7d494ac3 <unknown>\n\n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py:229: WebDriverException",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "driver",
    "time" : {
      "start" : 1728610933686,
      "stop" : 1728610934462,
      "duration" : 776
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "clear_allure_results_folder",
    "time" : {
      "start" : 1728610900918,
      "stop" : 1728610900918,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "contributors_page_open",
    "time" : {
      "start" : 1728610934462,
      "stop" : 1728610934884,
      "duration" : 422
    },
    "status" : "broken",
    "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x5570954ad02a <unknown>\n#1 0x5570951935e0 <unknown>\n#2 0x55709518b7eb <unknown>\n#3 0x55709517ba09 <unknown>\n#4 0x55709517d744 <unknown>\n#5 0x55709517bcbd <unknown>\n#6 0x55709517b52f <unknown>\n#7 0x55709517b422 <unknown>\n#8 0x557095179493 <unknown>\n#9 0x557095179ada <unknown>\n#10 0x557095195f41 <unknown>\n#11 0x5570952278c5 <unknown>\n#12 0x557095207b22 <unknown>\n#13 0x557095226d7d <unknown>\n#14 0x5570952078c3 <unknown>\n#15 0x5570951d56b3 <unknown>\n#16 0x5570951d668e <unknown>\n#17 0x557095477a2b <unknown>\n#18 0x55709547b9b1 <unknown>\n#19 0x557095464225 <unknown>\n#20 0x55709547c532 <unknown>\n#21 0x55709544938f <unknown>\n#22 0x55709549bf28 <unknown>\n#23 0x55709549c0f3 <unknown>\n#24 0x5570954abe7c <unknown>\n#25 0x7f2d7d494ac3 <unknown>\n\n",
    "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 902, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/tests/conftest.py\", line 28, in contributors_page_open\n    driver.get(MainPageLinks.URL_CONTRIBUTORS_PAGE)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 353, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
    "steps" : [ {
      "name" : "Open page: https://brainup.site/contributors",
      "time" : {
        "start" : 1728610934462,
        "stop" : 1728610934884,
        "duration" : 422
      },
      "status" : "broken",
      "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x5570954ad02a <unknown>\n#1 0x5570951935e0 <unknown>\n#2 0x55709518b7eb <unknown>\n#3 0x55709517ba09 <unknown>\n#4 0x55709517d744 <unknown>\n#5 0x55709517bcbd <unknown>\n#6 0x55709517b52f <unknown>\n#7 0x55709517b422 <unknown>\n#8 0x557095179493 <unknown>\n#9 0x557095179ada <unknown>\n#10 0x557095195f41 <unknown>\n#11 0x5570952278c5 <unknown>\n#12 0x557095207b22 <unknown>\n#13 0x557095226d7d <unknown>\n#14 0x5570952078c3 <unknown>\n#15 0x5570951d56b3 <unknown>\n#16 0x5570951d668e <unknown>\n#17 0x557095477a2b <unknown>\n#18 0x55709547b9b1 <unknown>\n#19 0x557095464225 <unknown>\n#20 0x55709547c532 <unknown>\n#21 0x55709544938f <unknown>\n#22 0x55709549bf28 <unknown>\n#23 0x55709549c0f3 <unknown>\n#24 0x5570954abe7c <unknown>\n#25 0x7f2d7d494ac3 <unknown>\n\n",
      "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/tests/conftest.py\", line 28, in contributors_page_open\n    driver.get(MainPageLinks.URL_CONTRIBUTORS_PAGE)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 353, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ {
        "name" : "driver",
        "value" : "<selenium.webdriver.chrome.webdriver.WebDriver (session=\"2b0bebe0fc7829f874c2d38e54d6945f\")>"
      } ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 1,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  } ],
  "afterStages" : [ {
    "name" : "driver::0",
    "time" : {
      "start" : 1728610934915,
      "stop" : 1728610934977,
      "duration" : 62
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "epic",
    "value" : "The Contributors Page"
  }, {
    "name" : "parentSuite",
    "value" : "tests"
  }, {
    "name" : "suite",
    "value" : "contributors_page_test"
  }, {
    "name" : "subSuite",
    "value" : "TestContributorsPage"
  }, {
    "name" : "host",
    "value" : "fv-az740-570"
  }, {
    "name" : "thread",
    "value" : "2375-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.contributors_page_test"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 3,
        "skipped" : 0,
        "passed" : 26,
        "unknown" : 0,
        "total" : 29
      },
      "items" : [ {
        "uid" : "425758daf19db996",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/58//#testresult/425758daf19db996",
        "status" : "passed",
        "time" : {
          "start" : 1728524640874,
          "stop" : 1728524642014,
          "duration" : 1140
        }
      }, {
        "uid" : "c551c425a9e3c19f",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/57//#testresult/c551c425a9e3c19f",
        "status" : "passed",
        "time" : {
          "start" : 1728438212360,
          "stop" : 1728438213506,
          "duration" : 1146
        }
      }, {
        "uid" : "c1515bb9e5726155",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/56//#testresult/c1515bb9e5726155",
        "status" : "passed",
        "time" : {
          "start" : 1728351876096,
          "stop" : 1728351877333,
          "duration" : 1237
        }
      }, {
        "uid" : "7beb3454e0387dc6",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/55//#testresult/7beb3454e0387dc6",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find elements by locator ('css selector', '.team-member img')",
        "time" : {
          "start" : 1728265829019,
          "stop" : 1728265849267,
          "duration" : 20248
        }
      }, {
        "uid" : "5da56c130047b39d",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/54//#testresult/5da56c130047b39d",
        "status" : "passed",
        "time" : {
          "start" : 1728179431221,
          "stop" : 1728179432372,
          "duration" : 1151
        }
      }, {
        "uid" : "70fa22b4e84b9e53",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/53//#testresult/70fa22b4e84b9e53",
        "status" : "passed",
        "time" : {
          "start" : 1728092608261,
          "stop" : 1728092609408,
          "duration" : 1147
        }
      }, {
        "uid" : "3653e230705f03ac",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/52//#testresult/3653e230705f03ac",
        "status" : "passed",
        "time" : {
          "start" : 1728006313050,
          "stop" : 1728006314216,
          "duration" : 1166
        }
      }, {
        "uid" : "ce0dd5e479d17d26",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/51//#testresult/ce0dd5e479d17d26",
        "status" : "passed",
        "time" : {
          "start" : 1727919902499,
          "stop" : 1727919903760,
          "duration" : 1261
        }
      }, {
        "uid" : "f68a908e662a1468",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/50//#testresult/f68a908e662a1468",
        "status" : "passed",
        "time" : {
          "start" : 1727833438278,
          "stop" : 1727833439497,
          "duration" : 1219
        }
      }, {
        "uid" : "dd1d3bea4001e5a7",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/49//#testresult/dd1d3bea4001e5a7",
        "status" : "passed",
        "time" : {
          "start" : 1727747574492,
          "stop" : 1727747575652,
          "duration" : 1160
        }
      }, {
        "uid" : "a5f7a6006e7c805e",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/48//#testresult/a5f7a6006e7c805e",
        "status" : "passed",
        "time" : {
          "start" : 1727660855532,
          "stop" : 1727660856711,
          "duration" : 1179
        }
      }, {
        "uid" : "e0cccd8f66e2f7d0",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/47//#testresult/e0cccd8f66e2f7d0",
        "status" : "passed",
        "time" : {
          "start" : 1727574663903,
          "stop" : 1727574665072,
          "duration" : 1169
        }
      }, {
        "uid" : "55da2ac0c1954285",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/46//#testresult/55da2ac0c1954285",
        "status" : "passed",
        "time" : {
          "start" : 1727487909698,
          "stop" : 1727487910838,
          "duration" : 1140
        }
      }, {
        "uid" : "c2de24da2a0477e7",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/45//#testresult/c2de24da2a0477e7",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find elements by locator ('css selector', '.team-member img')",
        "time" : {
          "start" : 1727401496869,
          "stop" : 1727401517185,
          "duration" : 20316
        }
      }, {
        "uid" : "433a9fe6862cd6cf",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/44//#testresult/433a9fe6862cd6cf",
        "status" : "passed",
        "time" : {
          "start" : 1727315042559,
          "stop" : 1727315043716,
          "duration" : 1157
        }
      }, {
        "uid" : "fa8372a8c08710fa",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/43//#testresult/fa8372a8c08710fa",
        "status" : "passed",
        "time" : {
          "start" : 1727228731481,
          "stop" : 1727228732638,
          "duration" : 1157
        }
      }, {
        "uid" : "de9b1fd8fc1b21e2",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/42//#testresult/de9b1fd8fc1b21e2",
        "status" : "passed",
        "time" : {
          "start" : 1727142284461,
          "stop" : 1727142285597,
          "duration" : 1136
        }
      }, {
        "uid" : "476ca8f3fdc465d3",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/41//#testresult/476ca8f3fdc465d3",
        "status" : "passed",
        "time" : {
          "start" : 1727055835014,
          "stop" : 1727055836116,
          "duration" : 1102
        }
      }, {
        "uid" : "7909ac6e981d6590",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/40//#testresult/7909ac6e981d6590",
        "status" : "passed",
        "time" : {
          "start" : 1726969761107,
          "stop" : 1726969762229,
          "duration" : 1122
        }
      }, {
        "uid" : "fcba3aba5acf22e9",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/39//#testresult/fcba3aba5acf22e9",
        "status" : "passed",
        "time" : {
          "start" : 1726882767223,
          "stop" : 1726882768293,
          "duration" : 1070
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "5788feeef3cc9b73.json",
  "parameterValues" : [ ]
}